### ISSUE TITLE
Day 2 Operations: Adding new Teleport Resources to existing Teleport node (a How To)

### ISSUE THREAD

**Issue by Valien** [gravitational/teleport#22540]
## Applies To

Nothing. This is new.

## Details

Something I think our docs are lacking is a simple guide for adding additional services to an existing node. All of the guides assume you are starting from scratch and go through details of installing Teleport, etc.

Even our wizards within our UI has the same flow.

Would love to see a wizard/guide that details how to add a new Teleport service to an existing fleet of X.

Example:

1. SSH node wizard is great but what if a user wants to then add an App?
2. They use the App wizard and it’s like starting all over. No easy way to flow from one to another (be cool to have a - would you like to add this app to an existing node? option)
3. So using the UI Wizard is useless and you end up resorting to going to the CLI with a tctl tokens add etc and manually updating the `teleport.yaml` file, etc.
4. But users might not realize this and can cause issues due to lack of solid docs around this.

TL;DR:

Have a guide/docs page that details adding a new Teleport service (`app`, `k8s`, `db`, etc) to an existing managed Teleport node. Which involves creating a new join token, and modifying the teleport.yaml file.

**Comment by webvictim** [comment]
In general I think that educating people about Teleport's "one process running multiple services" model is something we could do better. It would make it a little more obvious how to expand on an existing installation.

**Comment by ptgott** [comment]
Related: https://github.com/gravitational/teleport/issues/19007

**Comment by ptgott** [comment]
#27703 proposes adding a top-level docs section on how Teleport agents work and how to deploy them, so we could add this guide there.
